src/core/components/NavMenu.tsx

```diff
@@ -16,6 +19,8 @@ export const NavMenu = (): JSX.Element => {
     return isActiveRoute ? activeRouteClassName : '';
   };
 
+  const logout = () => dispatch(clearUserDate());
```

Comment:
Just to make sure it's 100% clear that we erase user data or date? USER_DATA_CLEARED: clearUserDate.

Reply:
Typo, fixed!